### DISCUSSION TITLE
Téléinfo Linky

### DISCUSSION THREAD

**Discussion by fdaubercy** [arendst/Tasmota#15439] [Ideas]
Bonjour.

Je cherche a savoir s'il est possible d'obtenir les données de 2 compteurs Linky différents.
Chacun des 2 compteurs étant connecté à un port différent de l'ESP32.

Je n'arrive pas à le paramétrer dans l'onglet 'module'.

Merci
Frédéric

**Comment by barbudor** [comment]
Bonjour Frédéric
Ce forum est en anglais. Si tu préfèe le français, tu peux rejoindre le forum de Charles : https://community.ch2i.eu/ 
En attendant la réponse en anglais

As of today the Teleinfo driver only supports 1 Linky.
The only solution I see would be to write a 2nd driver in Berry to handle the 2nd meter.
